examples/src/cross-contract-call.js

```diff
@@ -1,9 +1,14 @@
-import { NearContract, NearBindgen, call, view, near, bytes } from 'near-sdk-js'
+import { NearBindgen, call, view, initialize, near, bytes } from 'near-sdk-js'
 
-@NearBindgen
-class OnCall extends NearContract {
-    constructor({ statusMessageContract }) {
-        super()
+@NearBindgen({ requireInit: true })
+class OnCall {
+    constructor() {
+        this.personOnCall = ''
+        this.statusMessageContract = ''
+    }
```

Comment:
Since `requireInit` , can we omit the constructor?

Reply:
In this case, it is not possible to deserialize it since we do not know the list of properties.

Reply:
I see!